src/styles/main-card.scss

```diff
@@ -0,0 +1,22 @@
+.main {
```

Comment:
give a proper name for the file
![image](https://user-images.githubusercontent.com/99331793/198036617-c5748d3d-8a98-493c-b1de-1ecfede6094e.png)